src/DrupalConsoleStack.php

```diff
@@ -0,0 +1,698 @@
+<?php
+
+namespace DigipolisGent\Robo\Task\DrupalConsole;
+
+use Robo\Common\CommandArguments;
+use Robo\Task\CommandStack;
+
+/**
+ * Runs Drupal Console commands in stack. You can use `stopOnFail()` to point that stack should be terminated on first fail.
+ * You can define global options for all commands (like Drupal root and uri).
+ * The option --yes is always set, as it makes sense in a task runner.
+ *
+ * ``` php
+ * $this->taskDrupalConsoleStack()
+ *     ->drupalRootDirectory('/var/www/html/some-site')
+ *     ->uri('sub.example.com')
+ *     ->maintenanceOn(true)
+ *     ->updateDb()
+ *     ->configImport()
+ *     ->maintenanceOff()
+ *     ->run();
+ * ```
+ *
+ * Example site install command:
+ *
+ * ``` php
+ * $this->taskDrushStack()
+ *   ->siteName('Site Name Mysql')
+ *   ->siteMail('site-mail2@example.com')
+ *   ->langcode('fr')
+ *   ->accountMail('mail2@example.com')
+ *   ->accountName('admin-user')
+ *   ->accountPass('passw')
+ *   ->dbPrefix('drupal_')
+ *   ->dbType('mysql')
+ *   ->dbHost('localhost')
+ *   ->dbName('testdb')
+ *   ->dbUser('dbuser')
+ *   ->dbPass('testdbpw')
+ *   ->siteInstall('standard')
+ *   ->run();
+ * ```
+ */
+class DrupalConsoleStack extends CommandStack
+{
+
+    use CommandArguments;
+
+    /**
+     * Verbosity levels:
+     * 1 for normal output, 2 for more verbose output, and 3 for debug.
+     */
+    const VERBOSITY_LEVEL_NORMAL = 1;
+    const VERBOSITY_LEVEL_VERBOSE = 2;
+    const VERBOSITY_LEVEL_DEBUG = 3;
+
+    /**
+     * Pass arguments to the executable; applies to the next command only.
```

Comment:
s/Pass arguments to/Arguments for/
s/applies/apply/ (as in "The arguments apply [...]", not "The arguments applies [...]"